### PR TITLE
Homepage likes and items

### PR DESCRIPTION
**Pull request by odaialazzeh**
- [ ] Update CSS file and add styles to home items and likes.
- [ ] Update HTML file.
- [ ] Add a module to add likes and displays them.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
       <div class="logo"><h1>My <span>Meals</span></h1></div>
       <nav>
         <ul>
-          <li id="meals">Meals</li>
+          <li id="meals"></li>
```
Comment (SergSkol) [inline src/index.html:16]: Thank you for your commit. I propose to keep  this menu option for future development. We can add few pages and navigate to Categories, Ingridients and back to Meals.

File: src/module/add-like.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+async function addLike(id) {
+  fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tnE2k6P5BdZ2HCTjbd0V/likes', {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+      likes: 1,
```
Comment (SergSkol) [inline src/module/add-like.js:6]: No need to add "likes:1" to POST, it will automatically add only1 like.

**Review by SergSkol — APPROVED** [review]
Thank you for your comminment! It's great! You can merge it already. But I have few optional recommendations, please find it in comments.

File: src/module/add-like.js (hunk 2 above)
Comment (odaialazzeh) [reply src/module/add-like.js:6]: Thank you for the feedback, it was very helpful.👍

**Pull request merged**